tasks/shelter/shelter-part1.md

```diff
@@ -1,142 +1,139 @@
 
 ## Перечень ссылочных связей
-#### На странице Main
+### На странице Main
 1. На страницу `Pets` ведут:
-   - элемент навигации `Our Pets`
-   - кнопка `Get to know the rest` в блоке **Our Friends**
-2. На блок **Help** текущей страницы (якорная ссылка) ведут:
-   - элемент навигации `Help the shelter`
-3. На блок **Our Friends** текущей страницы (якорная ссылка) ведут:
-   - кнопка `Make a Friend` в блоке **Not only**
+   - элемент навигации `Our Pets`;
+   - кнопка `Get to know the rest` в блоке `Our Friends`.
+2. На блок `Help` текущей страницы (якорная ссылка) ведут:
+   - элемент навигации `Help the shelter`.
+3. На блок `Our Friends` текущей страницы (якорная ссылка) ведут:
+   - кнопка `Make a Friend` в блоке `Not only`.
 4. На футер текущей страницы (якорная ссылка) ведут:
-   - элемент навигации `Contacts`
+   - элемент навигации `Contacts`.
 5. Пустая ссылка:
-   - логотип
-   - номер банковской карты в блоке **In addition**
+   - логотип;
+   - номер банковской карты в блоке `In addition`.
 6. Другие варианты поведения:
-   - в футере при нажатии на email или его иконку должен открываться почтовый сервис
-   - в футере при нажатии на телефон или его иконку должен открываться набор номера
-   - в футере при нажатии на локацию должна открываться страница с google maps в отдельном окне или вкладке браузера с любой локацией на ваш выбор
+   - в футере при нажатии на email или его иконку должен открываться почтовый сервис;
+   - в футере при нажатии на телефон или его иконку должен открываться набор номера;
+   - в футере при нажатии на локацию должна открываться страница с google maps в отдельном окне или вкладке браузера с любой локацией на ваш выбор.
 
-#### На странице Pets
+### На странице Pets
 1. На страницу `Main` ведут:
-   - элемент навигации `About the shelter`
-   - логотип в хедере
-2. На блок **Help** страницы `Main` (якорная ссылка) ведут:
-   - элемент навигации `Help the shelter`
+   - элемент навигации `About the shelter`;
+   - логотип в хедере.
+2. На блок `Help` страницы `Main` (якорная ссылка) ведут:
+   - элемент навигации `Help the shelter`.
 3. На футер текущей страницы (якорная ссылка) ведут:
-   - элемент навигации `Contacts`
+   - элемент навигации `Contacts`.
 4. Другие варианты поведения:
-   - в футере при нажатии на email или его иконку должен открываться почтовый сервис
-   - в футере при нажатии на телефон или его иконку должен открываться набор номера
-   - в футере при нажатии на локацию должна открываться страница с google maps в отдельном окне или вкладке браузера с любой локацией на ваш выбор
+   - в футере при нажатии на email или его иконку должен открываться почтовый сервис;
+   - в футере при нажатии на телефон или его иконку должен открываться набор номера;
+   - в футере при нажатии на локацию должна открываться страница с google maps в отдельном окне или вкладке браузера с любой локацией на ваш выбор;
```

Comment:
why here ';' at the end as accordingly to other changes should be '.' (the last in this list)